.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
You should have a new line at the end of this file

Reply:
New line added